src/mpid/ch4/include/mpidpre.h

```diff
@@ -187,11 +191,9 @@ typedef struct MPIDIG_req_t {
 typedef struct {
 #ifndef MPIDI_CH4_DIRECT_NETMOD
     int is_local;
-#endif
     /* Anysource handling. Netmod and shm specific requests are cross
      * referenced. This must be present all of the time to avoid lots of extra
      * ifdefs in the code. */
-#ifndef MPIDI_CH4_DIRECT_NETMOD
     struct MPIR_Request *anysource_partner_request;
 #endif
 
```

Comment:
The above change seems to be simple cleanup and can be moved to a sperate commit.

Reply:
Done